.github/workflows/review.yml

```diff
@@ -1,4 +1,4 @@
-name: AI Code Reviewer
+name: GPTcode-reviwer
```

Comment:
The name 'GPTcode-reviwer' contains a typo. Consider correcting it to 'GPTcode-reviewer'.